index.html

```diff
@@ -285,9 +285,17 @@ <h2>Coordination</h2>
       <div>
         <h3 id="w3c-coordination">W3C Groups</h3>
 
-        <p>To promote interoperability with the overall web platform, and for all Recommendation-track documents, the
-        Working Group SHALL seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal
-        review</a> with the W3C Horizontal Groups listed below.</p>
+
+
+        <p>For all specifications, this Working Group will seek <a
+          href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal review</a> for
+          accessibility, internationalization, performance, privacy, and security with the relevant Working and
+          Interest Groups, and with the <a href="https://www.w3.org/2001/tag/"
```

Comment:
As suggested by @nigelmegitt at https://github.com/w3c/strategy/issues/177#issuecomment-498174148, the comment at https://github.com/w3c/strategy/issues/177#issuecomment-497142640 can be addressed by replacing "the
        Working Group SHALL seek horizontal review with the W3C Horizontal Groups listed below" with something along the lines of "the
        Working Group SHALL seek review from W3C horizontal groups and other relevant W3C groups, including:"